service/Makefile.am

```diff
@@ -64,6 +64,19 @@ EXTRA_DIST = .gitignore \
              configure.ac \
              contrib/json11/LICENSE.txt \
              contrib/json11/README.md \
+             debian/copyright \
+             debian/geopm-service.install \
+             debian/control \
+             debian/python3-geopmdpy.install \
+             debian/libgeopmd-dev.dirs \
+             debian/libgeopmd1.install \
+             debian/libgeopmd1.symbols \
+             debian/geopm-service.postinst \
+             debian/libgeopmd-dev.install \
+             debian/changelog \
+             debian/changelog.in \
+             debian/geopm-service.dirs \
+             debian/rules \
```

Comment:
ABC order, and geopm.service seems to be absent from the distribution tarball.  Perhaps the symbolic link is not required if it exists in the base directory?